jenkins/kola/aws.sh

```diff
@@ -44,6 +44,33 @@ if [[ "${AWS_AMI_ID}" == "" ]]; then
   echo "Created new AMI ${AWS_AMI_ID} (will be removed after testing)"
 fi
 
+# Run the cl.internet test on multiple machine types only if it should run in general
```

Comment:
Should the `cl.internet` test be filtered out from `KOLA_TESTS` to avoid running it twice?

Reply:
(We want to run the cl.internet service on all instance types)

Reply:
Ok, nevermind, now I have noticed that the extra cl.internet run is done of potentially different instance types.